zfs_snapshot.md

```diff
@@ -0,0 +1,36 @@
+# ZFS snapshot
+## Update secrect file
+```
+export POOL_NAME=
+export USER=
```

Comment:
There are common well-known PostgreSQL environment variables you can use. Please use them as they are the same across different commands (psql, pg_dump).

You can find them here:

https://www.postgresql.org/docs/current/libpq-envars.html